data/base-hunting.yaml

```diff
@@ -229,6 +229,15 @@ hunting_zones:
   - 8113
   - 8108
   - 8110
+    # https://elanthipedia.play.net/Granite_gargoyle 95-125
+  granite_gargoyle
```

Comment:
You need a colon (`:`) after the hunting zone name.

As well, this name is too close to the one that already exists in the file. What do you think about `granite_gargoyles_shard`?